File: Upcoder/Other/Vẽ HCN rỗng/LTCB_112b.cs
using System;
namespace HCN
{
    public class Program
    {
        static void Main(string[]args)
        {
            string []input = Console.ReadLine().Split(' ');
            int n = int.Parse(input[0]);
            int m = int.Parse(input[1]);
            for(int i = 0;i < m; i++)
            {
                for(int j = 0; j < n; j++)
                {
                    if (i == 0 || i == m - 1) 
                    {
                        Console.Write("*");
                    }
                    else 
                    {
                        if (j == 0 || j == n - 1) 
                            Console.Write("*");
                        else
                            Console.Write(" ");
                    }
                }
                Console.WriteLine("");
            }
        }
    }
}

File: Upcoder/Other/Vẽ HCN rỗng/LTCB_112b.py
n, m = map(int, input().split())
for i in range(0, m):
    for j in range (0, n):
        if(i == 0 or i == m - 1):
            print("*", end="")
        else:
            if(j == 0 or j == n - 1):
                print("*", end="")
            else:
                print(" ", end="")
    print()